debezium-core/src/main/java/io/debezium/pipeline/metrics/Metrics.java

```diff
@@ -28,12 +28,14 @@
 public abstract class Metrics implements DataChangeEventListener, ChangeEventSourceMetricsMXBean {
 
     protected final AtomicLong totalNumberOfEventsSeen = new AtomicLong();
+    protected final AtomicLong numberOfEventsSkipped = new AtomicLong();
     protected final AtomicLong lastEventTimestamp = new AtomicLong(-1);
 
     private final String contextName;
     protected final Clock clock;
-    private final CdcSourceTaskContext taskContext;
+    protected final CdcSourceTaskContext taskContext;
     private volatile ObjectName name;
+    private volatile String lastEvent;
```

Comment:
WDYT about exposing this only after opting in via a new connector option? It seems like not everybody might want to expose the raw event in their metrics.

Reply:
This does not expose the content of the event - just metadata so I kept it. It mirros the existing MySQL metric.